code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -898,23 +901,64 @@
 	overlay_state = "tags"
 	drop_sound = 'sound/items/drop/accessory.ogg'
 	pickup_sound = 'sound/items/pickup/accessory.ogg'
-	var/can_be_broken = FALSE
+	gender = PLURAL
+	var/can_be_broken = TRUE
 	var/separated = FALSE
 	var/tag_type = /obj/item/dogtag
 
+	var/dogtags_name = DOGTAGS_UNSET
+	var/dogtags_faction = DOGTAGS_UNSET
+	var/dogtags_religion = DOGTAGS_UNSET
+	var/dogtags_btype = DOGTAGS_UNSET
+
+/obj/item/dogtag
+	name = "dogtag"
+	desc = "An engraved metal identification tag, recovered from the dogtags of a deceased individual."
+	icon = 'icons/clothing/accessories/dogtags.dmi'
+	icon_state = "tag"
+	w_class = ITEMSIZE_TINY
+
+	var/tag_name = DOGTAGS_UNSET
+	var/tag_faction = DOGTAGS_UNSET
+	var/tag_religion = DOGTAGS_UNSET
+	var/tag_btype = DOGTAGS_UNSET
+
 /obj/item/clothing/accessory/dogtags/get_mask_examine_text(mob/user)
 	return "around [user.get_pronoun("his")] neck"
 
 /obj/item/clothing/accessory/dogtags/attack_self(mob/user)
-	if(can_be_broken)
+	if(use_check_and_message(user))
+		return
+
+/obj/item/clothing/accessory/dogtags/attack_self(var/mob/living/carbon/human/user = user)
```

Comment:
```suggestion
/obj/item/clothing/accessory/dogtags/attack_self(var/user)
```

Reply:
`var/user` won't compile. All the information required by the dog tags is listed in `human_defines.dm`.
![image](https://github.com/Aurorastation/Aurora.3/assets/44721288/faaa11b5-0f03-45b1-b018-c9545bdfdc5e)